rktl_planner/launch/simple_agent.launch

```diff
@@ -0,0 +1,19 @@
+<launch>
+    <arg name="agent_name" default="agent0"/>
+    <arg name="car_name" default="car0"/>
+
+    <group ns="agents/$(arg agent_name)">
+        <!-- Path Follower -->
+        <node pkg="rktl_planner" type="path_follower" name="path_" output="screen">
+            <rosparam command="load" file="$(find rktl_planner)/config/path_follower.yaml" />
+            <param name="car_name" value="$(arg car_name)" />
```

Comment:
I dislike putting a space at the end here. Whichever way we go, I think it would be worth standardizing.